data/raw/f_3345_junda_james.py

```diff
@@ -0,0 +1,102 @@
+import cv2
+import os
+from matplotlib import pyplot as plt
+
+def f_3345(image_path='image.jpg', histogram_path='histogram.png'):
+    """
+    Read an image, create a histogram of the image pixel intensities, save the histogram as a PNG file, 

```

Comment:
mention the possible size, as you use `cv2.calcHist([img], [0], None, [256], [0, 256])`